code/__HELPERS/matrices.dm

```diff
@@ -24,6 +24,9 @@
 		animate(transform = matrices[i], time = speed)
 		//doesn't have an object argument because this is "Stacking" with the animate call above
 		//3 billion% intentional
+	if(isliving(src))
```

Comment:
this is nowhere near acceptable and a pretty egregious violation of OOP.

Reply:
Whoops, didn't even notice this was a helper - I'll fix that in a moment